src/components/DeckMechanisms/index.js

```diff
@@ -14,6 +14,12 @@ const DAWNSPARKS_HITS = 0.71
 const FROZEN_CORE_STAYS = 0.5
 const AHMI_RETURNS = 0.5
 
+const END_TURN_CARD = {
+  id: 'END',
+  name: 'End Turn',
+  image: '/assets/images/mana.png',
+}
+
```

Comment:
Temporary mana icon to indicate the end turn button has been pressed. This could be used as one of the indicators to make the turn transition clearer

Reply:
I like the way you think. Maybe we could introduce a small border between turns instead? That will also make it possible to display extra cards since we wouldn’t have the mana placeholder.